src/main/java/dto/orderitem/OrderRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package dto.orderitem;
+
+import jakarta.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.Length;
+
+public record OrderRequestDto(
+        @NotNull @Length(max = 255)
```

Comment:
It's better to use Size annotation, Length is too specific Hibernate annotation
Minor